### ISSUE TITLE
FreeBSD Support

### ISSUE THREAD

**Issue by colton22** [tmc/keyring#18]
Hey all!

I am trying to compile this project for FreeBSD (everything works well in Linux and Darwin) but FreeBSD I keep getting an error as follows:

```
$> env GOOS=freebsd GOARCH=amd64 go build keyring.go 
# command-line-arguments
./keyring.go:27:40: undefined: initializeProvider
$>
```

Is there any desire to support FreeBSD on this project?

**Comment by tmc** [comment]
@colton22 thanks for your comment. I don't have a freebsd box nearby so it's hard for me to work on this but I'd be more than happy to accept a patch!

**Comment by colton22** [comment]
If only I knew more of it :) I am not on bsd much but mainly as a Jumphost where a lot of other tools lay. I will see what I can find.

**Comment by paulquerna-okta** [comment]
Fixed by this PR: https://github.com/tmc/keyring/pull/19

**Comment by colton22** [comment]
Awesome! Thank you!! 

Closing this ticket.

**Comment by jayfaulkner-okta** [comment]
@colton22 this ticket was closed, but #19 was never merged -- can you revisit?

Thanks!
